helpdesk/management/commands/get_email.py

```diff
@@ -5,18 +5,21 @@
 (c) Copyright 2008 Jutda. All Rights Reserved. See LICENSE for details.
 
 scripts/get_email.py - Designed to be run from cron, this script checks the
-                       POP and IMAP boxes defined for the queues within a
+                       POP and IMAP boxes, or a local mailbox directory,
+                       defined for the queues within a
                        helpdesk, creating tickets from the new messages (or
                        adding to existing tickets if needed)
 """
-from __future__ import print_function
 
 import email
 import imaplib
 import mimetypes
 import poplib
 import re
 import socket
+import six
```

Comment:
Can we use [django.utils.six](https://docs.djangoproject.com/en/1.7/topics/python3/#module-django.utils.six) instead? This would prevent the additional dependency.